osu.Game/GameModes/Play/BeatmapGroup.cs

```diff
@@ -0,0 +1,246 @@
+//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Beatmaps;
+using osu.Game.Database;
+using osu.Framework.Graphics.Primitives;
+using OpenTK;
+using System.Linq;
+using osu.Framework.Graphics.Transformations;
+using osu.Framework.Input;
+using OpenTK.Graphics;
+using osu.Game.Beatmaps.IO;
+using osu.Framework.Graphics.Textures;
+using System.Threading.Tasks;
+
+namespace osu.Game.GameModes.Play
+{
+    class BeatmapGroup : Container
```

Comment:
Even though it really consists of multiple discrete panels? This control has ownership over the set's panel and the difficulty panels as well.